scitasexternal/packages/molpro/package.py

```diff
@@ -5,24 +5,25 @@
 
 from spack import *
 import os
-import subprocess
 
 
 class Molpro(Package):
     """Molpro is an ab initio program for electronic structure calculations."""
 
     homepage = "http://www.molpro.net"
-    url      = "fake_url.tar.gz"
+    url      = 'file://%s/molpro-2019.2.tar.gz' % os.getcwd()
```

Comment:
Will this work for `2015.1`?
If not we could use some variant of `self.version.up_to(2)`.

Reply:
It will. I just tested it, to make sure. Since both versions have that exact numbering scheme, spack can find them, and we already put the files in the mirror anyway.